pkg/statistics/handle/syncload/stats_syncload.go

```diff
@@ -238,8 +238,21 @@ func (s *statsSyncLoad) SubLoadWorker(sctx sessionctx.Context, exit chan struct{
 		if err != nil {
 			switch err {
 			case errExit:
+				statslogutil.StatsLogger().Info("SubLoadWorker: exits now because the domain is closed.")
 				return
 			default:
+				const msg = "SubLoadWorker: failed to handle one task"
+				if task != nil {
+					statslogutil.SingletonStatsErrVerboseSamplerLogger().Warn(msg,
```

Comment:
Could we simplify the name？ The log usually is Singleton so we don't need to enhance it when naming

Reply:
The singleton here does not emphasize that the logger is a singleton, but rather that the sample log must be a singleton; otherwise, the sampling mechanism will fail. I remember this naming convention was decided last year in a discussion between you, Lingxiang, and me.

Reply:
Maybe we can remove the singleton prefix here, as it seems more like an implementation detail. We might not need to expose it to the caller.

Reply:
I have removed the singleton prefix.